ibm/resource_ibm_satellite_location.go

```diff
@@ -123,6 +123,13 @@ func resourceIBMSatelliteLocation() *schema.Resource {
 				Computed:    true,
 				Description: "ID of the resource group.",
 			},
+			"tags": {
+				Type:     schema.TypeSet,
```

Comment:
remove this changes its handled as part of other pr

Reply:
Removed tags